x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -658,4 +658,35 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'windows.advanced.memory_protection.shellcode_enhanced_pe_parsing',
+    first_supported_version: '7.15',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.memory_protection.shellcode_enhanced_pe_parsing',
+      {
+        defaultMessage:
+          "A value of 'false' disables enhanced parsing of PEs found within shellcode payloads. Default: true.",
```

Comment:
Just curious, what is PE?

Reply:
Google says portable executable. I am not versed on the topic. Consider this is an advance option, users using these should know what they are doing.

Reply:
Right 👍  Not sure if we should use the expanded jargon vs just PE. 🤔